### PR TITLE
No ticket update safe links

### PR DESCRIPTION
**Pull request by jdf400-rapid7**
## Proposed Changes
Update ATP safelinks logic for returning non safelinks
### Description

Describe the proposed changes:

  -

### Testing

Any testing information goes here.

## PR Requirements

Developers, verify you have completed the following items by checking them off:

### Style

Review the [style guide](https://komand.github.io/python/style.html)

- [ ] For dependencies, pin [OS package](https://komand.github.io/python/style.html#dockerfile) and [Python package](https://komand.github.io/python/style.html#requirements-txt) versions
- [ ] For security, set least privileged account with ``USER nobody`` in the ``Dockerfile`` when possible
- [ ] For size, use the [slim SDK images](https://komand.github.io/python/sdk.html#sdk-versions) when possible: ``komand/python-3-37-slim-plugin`` and ``komand/python-3-37-plugin``
- [ ] For error handling, use of [PluginException](https://komand.github.io/python/error_handling.html#plugin-exceptions) and [ConnectionTestException](https://komand.github.io/python/error_handling.html#connection-exceptions)
- [ ] For logging, use [self.logger](https://komand.github.io/python/sdk.html#logging)
- [ ] For docs, use [changelog style](https://komand.github.io/python/style.html#changelog)
- [ ] For docs, validate markdown with ``make validate`` which calls ``mdl`` to lint ``help.md``

### Functional Checklist
- [ ] Work fully completed
- [ ] Functional
  - [ ] Any new actions/triggers include JSON [test files](https://komand.github.io/python/style.html#tests) in the `tests/` directory created with `icon-plugin run -c sample $action > tests/$action.json`
  - [ ] Tests should all pass unless it's a negative test. Negative tests have a naming convention of `tests/$action_bad.json`
  - [ ] Unsuccessful tests should fail by raising an exception causing the plugin to die and an object should be returned on successful test
  - [ ] Add functioning test results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -T tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -T all --debug --jq` (use PR format at end)
  - [ ] Add functioning run results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -R tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -R all --debug --jq` (use PR format at end)

### Assessment

You must validate your work to reviewers:

1. Run `make validate` and make sure everything passes
2. Run the assessment tool: `icon-plugin run -A -R all -T all`. For single action validation: `icon-plugin run -A -R tests/my_action.json -T tests/my_action.json`
3. Copy (`icon-plugin ... | pbcopy`) and paste the output in **a new post** on this PR.
4. Add UI screenshot of the workflow used for testing
5. Add UI screenshot of the job output used for testing
6. Add UI screenshot of the artifact (See rules in UI Checklist) used for testing

### CODE REVIEW

**Comment by jdf400-rapid7** [comment]
INFO[0000] Running command:  docker run --rm -i rapid7/microsoft_atp_safe_links:1.0.3 run < tests/decode_nonsafelink.json 
rapid7/Microsoft Office 365 ATP Safe Links:1.0.3. Step name: decode
{"body": {"log": "rapid7/Microsoft Office 365 ATP Safe Links:1.0.3. Step name: decode\n", "status": "ok", "meta": {}, "output": {"result": "https://innovaccer.com/forms/unsubscribe/1b95755e-f23d-4f41-a198-79b0788b9a47/?email_1=amar@sunshinehh.com&amp;event=webinar\u2029"}}, "version": "v1", "type": "action_event"}

## Assessment
### Run

<details>

```
{
  "body": {
    "log": "rapid7/Microsoft Office 365 ATP Safe Links:1.0.3. Step name: decode\n",
    "meta": {},
    "output": {
      "result": "https://innovaccer.com/forms/unsubscribe/1b95755e-f23d-4f41-a198-79b0788b9a47/?email_1=amar@sunshinehh.com\u0026amp;event=webinar\u2029"
    },
    "status": "ok"
  },
  "type": "action_event",
  "version": "v1"
}

```

<summary>
docker run --rm -i rapid7/microsoft_atp_safe_links:1.0.3 run < tests/decode_nonsafelink.json
</summary>
</details>

### UI

Screenshots of the plugin being used in Komand for validating use go below this line.

#### Workflow Builder
image


#### Job
image

#### Artifact
image
image

File: microsoft_atp_safe_links/komand_microsoft_atp_safe_links/util/utils.py [hunk 1]
```diff
@@ -8,4 +8,4 @@ def decode_url(url: str) -> str:
     for param in split_url_params:
         if param.split("=")[0] == "url":
             return unquote(param.split("=")[1])
-    return ""
+    return url
```
Comment (jmcadams-r7) [inline microsoft_atp_safe_links/komand_microsoft_atp_safe_links/util/utils.py]: I think we can do more here to ensure it's a URL...however, the exit case in the action just returns the input as well...so eh...I can go either way.

File: microsoft_atp_safe_links/help.md [hunk 2]
```diff
@@ -40,10 +40,20 @@ Example output:
 
 ```
 {
-  "result":"https://aomediacodec.github.io/av1-spec/av1-spec.pdf"
+  "result": "https://innovaccer.com/forms/unsubscribe/1b95755e-..."
```
Comment (lmilby-r7) [inline microsoft_atp_safe_links/help.md]: revert

File: microsoft_atp_safe_links/tests/decode_nonsafelink.json [hunk 3]
```diff
@@ -3,10 +3,10 @@
         "action": "decode",
         "connection": null,
         "input": {
-            "url": "rapid7.com"
+            "url": "https://innovaccer.com/forms/unsubscribe/1b95755e-f23d-4f41-a198-79b0788b9a47/?email_1=blah&amp;event=webinar "
```
Comment (lmilby-r7) [inline microsoft_atp_safe_links/tests/decode_nonsafelink.json]: revert

**Comment by jdf400-rapid7** [comment]
It should be there https://github.com/rapid7/insightconnect-plugins/pull/349/files#diff-37a1f625752b23bd39ccddfd6fbc95caR71

**Pull request merged**